endpoints/openrtb2/amp_auction.go

```diff
@@ -182,13 +182,17 @@ func (deps *endpointDeps) AmpAuction(w http.ResponseWriter, r *http.Request, _ h
 		return
 	}
 
+	// #1712
+	secgpc := r.Header.Get("Sec-GPC")
```

Comment:
Nitpick: I don't recommend adding the issue id as a comment. Devs can always use git blame if they're like to track down where it comes from. 

Nitpick: Please consider naming it "secGPC" to better align with the header name.

Reply:
This is also updated now with your recommendations.